js/views/teamHistory.js

```diff
@@ -99,7 +98,16 @@ define(["dao", "globals", "ui", "core/player", "lib/bluebird", "lib/jquery", "li
 
         ko.computed(function () {
             ui.datatable($("#team-history-players"), 2, _.map(vm.players(), function (p) {
-                return [helpers.playerNameLabels(p.pid, p.name, p.injury, [], p.watch), p.pos, String(p.careerStats.gp), helpers.round(p.careerStats.min, 1), helpers.round(p.careerStats.pts, 1), helpers.round(p.careerStats.trb, 1), helpers.round(p.careerStats.ast, 1), helpers.round(p.careerStats.per, 1), helpers.round(p.careerStats.ewa, 1), p.hof, p.tid > g.PLAYER.RETIRED && p.tid !== vm.team.tid(), p.tid === vm.team.tid()];
+                var i, lastYr;
```

Comment:
This stuff shouldn't be done here. Instead, create a new property in the loop above called players[i].lastYear and then just use that variable here.